File: BookLibrary/Scripts/app/model/Person.js
Ext.define('App.model.Person',
{
    extend: 'Ext.data.Model',
    idProperty: 'Id',
    fields: [
        { name: 'Id', type: 'int', defaultValue: 0 },
        { name: 'Name', type: 'string' },
        { name: 'Surname', type: 'string' },
        {
            name: 'DateOfBirth', type: 'date', 
            dateFormat: 'MS',
            serialize: function (value, record) {
                //Convert date type that .NET can bind to DateTime
                var date = new Date(parseInt(value.substr(6)));
                return Ext.Date.format(date, "d-m-Y"); //Full Date Time
            }
        },
        { name: 'BooksData', type: '' } //типа для книг, которые должен
    ],
    proxy:
     {
         
         type: 'ajax',
         api:
         {
             read: 'Home/GetPersonsData'
         },
         reader:
         {
             type: 'json',
             root: 'persons',
             totalProperty: 'TotalCount'
         },
  
     }
});

File: BookLibrary/Models/Person.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace BookLibrary.Models
{
    public class Person
    {
        
        public int Id { get; set; }
        public string Name { get; set; }
        public string Surname { get; set; }
        public DateTime DateOfBirth { get; set; }

        public List<Book> BooksData { get; set; } 
    }
}